src/liboslexec/shadingsys.cpp

```diff
@@ -265,9 +265,26 @@ ShadingSystem::execute (ShadingContext &ctx, ShaderGroup &sas,
 
 
 const void*
-ShadingSystem::get_symbol (ShadingContext &ctx, ustring name, TypeDesc &type)
+ShadingSystem::get_symbol (ShadingContext &ctx, ustring layername,
```

Comment:
Makes sense, though in combination with the other fix -- the layers who's symbols you ask for should in theory have been run already.

Up to the user to declare upfront they will be reading certain symbols if they want them to be initialized properly.